src/django/api/templates/mail/facility_list_complete_body.html

```diff
@@ -120,6 +120,6 @@
     <p>
```

Comment:
You can test the processing complete email in development with
```
./scripts/manage batch_process -a notify_complete -l 16
```

On line 118 we include the info@ email in the body text. I would suggest changing it to data@ to harmonize with the from address.

Reply:
https://github.com/open-apparel-registry/open-apparel-registry/pull/2373/commits/9ddf0633170f9e9ca9b01a61bb935e6be3636ef0

Reply:
https://github.com/open-apparel-registry/open-apparel-registry/pull/2373/commits/a98520a98bdb5de9012ccac055f9994474334fb0